webcrypto/subtle_crypto_test.go

```diff
@@ -26,3 +27,47 @@ func TestSubtleDigest(t *testing.T) {
 
 	assert.NoError(t, gotScriptErr)
 }
+
+func TestSubtleCryptoGenerateKey(t *testing.T) {
+	t.Parallel()
+
+	t.Run("successes", func(t *testing.T) {
+		t.Parallel()
+
+		ts := newTestSetup(t)
+
+		gotScriptErr := ts.ev.Start(func() error {
+			program, err := CompileFile("./tests/generateKey", "successes.js")
+			require.NoError(t, err)
+
+			_, err = ts.rt.RunProgram(program)
+			require.NoError(t, err)
+
+			_, err = ts.rt.RunString(`run_test()`)
+
+			return err
+		})
+
+		assert.NoError(t, gotScriptErr)
+	})
+
+	t.Run("failures", func(t *testing.T) {
+		t.Parallel()
+
+		ts := newTestSetup(t)
+
+		gotScriptErr := ts.ev.Start(func() error {
+			program, err := CompileFile("./tests/generateKey", "failures.js")
+			require.NoError(t, err)
+
+			_, err = ts.rt.RunProgram(program)
+			require.NoError(t, err)
+
+			_, err = ts.rt.RunString(`run_test()`)
+
+			return err
+		})
+
+		assert.NoError(t, gotScriptErr)
+	})
+}
```

Comment:
Wouldn't this better with some kind of helper that just takes the files to run? Or runs all files within a directory like we(and goja) do for tc39 tests?